docs/customization.rst

```diff
@@ -118,6 +118,24 @@ Unlike, newer versions of mkdocs-material theme, this theme also supports the us
 
         :hero: Configuration options to personalize your site.
 
+.. confval:: hide
```

Comment:
Technically this isn't an `html_theme_options` dict (same applies to `hero`) so it might be better to have a separate directive and separate section of the documentation for sphinx metadata fields.

Reply:
Yeah that does seem confusing. And it doesn't seem explicitly clear that the metadata only applies to a single page.